.github/workflows/CD.yaml

```diff
@@ -25,6 +25,9 @@ on:
         type: string
         default: integration
 
+env:
+  GITHUB_USER: ${{ github.actor }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Warum gibt es hier neue envs die es in main nicht gibt?

Reply:
removed